src/ssh/cockpitsshrelay.c

```diff
@@ -466,6 +466,98 @@ static void cleanup_knownhosts_file (void)
 }
 
 /**
+ * machines_d_lookup_hostkey:
+ *
+ * Check if @host/@port is known anywhere in /etc/cockpit/machines.d/
+ *
+ * Returns: (transfer full): host key, or %NULL if no key was found.
+ * The returned value should be freed with g_free().
+ */
+static gchar* machines_d_lookup_hostkey (const gchar* host,
```

Comment:
In cockpit we put the function name at the beginning of the line, and the return value on the line above.